scripts/Makefile.lib

```diff
@@ -70,8 +70,10 @@ _c_flags       = $(filter-out $(CFLAGS_REMOVE_$(basetarget).o), $(orig_c_flags))
 orig_a_flags   = $(KBUILD_CPPFLAGS) $(KBUILD_AFLAGS) $(KBUILD_SUBDIR_ASFLAGS) \
                  $(asflags-y) $(AFLAGS_$(basetarget).o)
 _a_flags       = $(filter-out $(AFLAGS_REMOVE_$(basetarget).o), $(orig_a_flags))
-_cpp_flags     = $(KBUILD_CPPFLAGS) $(cppflags-y) $(CPPFLAGS_$(@F))
+_cpp_flags     = $(KBUILD_CPPFLAGS)  $(KBUILD_CFLAGS) $(cppflags-y) $(CPPFLAGS_$(@F))
```

Comment:
Isn't `_cpp_flags` supposed to be the C _preprocessor_ flags only? Usually it is `CXXFLAGS` and similar that contains C++-specific flags.

Reply:
You are right, I will make it more clear.

Reply:
Fixed in later commit

Reply:
Note that you should not add `$(KBUILD_CFLAGS)` then to `_cpp_flags` as `$(KBUILD_CFLAGS)` are flags for the C compiler and `_cpp_flags` are flags for the preprocessor.